common/app/views/support/package.scala

```diff
@@ -117,6 +117,9 @@ object `package` {
 object GUDateTimeFormat {
   def formatDateTimeForDisplay(date: DateTime, request: RequestHeader): String = {
     val edition = Edition(request)
+    formatDateTimeForDisplayGivenEdition(date: DateTime, edition: Edition)
```

Comment:
In this kind of case, for naming I normally encourage people to import the object, and then use that to namespace things. E.g.

    GUDateTimeFormat.forEdition(date: DateTime, edition: Edition)

if that makes sense? This is Go-inspired in fact.

Reply:
I decided to go with Mariot's suggestion but I see your point :)